worker/src/RTC/DtlsTransport.cpp

```diff
@@ -708,15 +734,19 @@ namespace RTC
 			goto error;
 		}
 
-		SSL_set_bio(this->ssl, this->sslBioFromNetwork, this->sslBioToNetwork);
-
 		// Set the MTU so that we don't send packets that are too large with no
 		// fragmentation.
-		// TODO: This is not honored, see issue:
-		//   https://github.com/versatica/mediasoup/issues/1100
 		SSL_set_mtu(this->ssl, DtlsMtu);
 		DTLS_set_link_mtu(this->ssl, DtlsMtu);
 
+		// We want to monitor OpenSSL write operations into our |sslBioToNetwork|
+		// buffer so we can immediately send those DTLS bytes (containing full DTLS
+		// messages, or valid DTLS fragment messages, or combination of them) to
+		// the endpoint, and hence we honor the configured DTLS MTU.
+		BIO_set_callback_ex(this->sslBioToNetwork, onSslBioOut);
```

Comment:
wasn't this an experimental function used to debug?

https://www.openssl.org/docs/man3.2/man3/BIO_set_callback_ex.html

Reply:
No.

> `BIO_set_callback_ex()` and `BIO_get_callback_ex()` set and retrieve the BIO callback. The callback is called during most high-level BIO operations. It can be used for debugging purposes to trace operations on a BIO or **to modify its operation**.